### PR TITLE
RABBITMQ: Set RABBITMQ_LOGS env variable default to empty

### PR DESCRIPTION
**Pull request by anthonyhaussman**
**Description of the change**

To setup `log.console`/`log.file` in RabbitMQ configuration, it is needed to set the environment variable `RABBITMQ_LOGS` to be empty.

It is needed to set JSON logs via the RabbitMQ configuration as discussed in https://github.com/bitnami/bitnami-docker-rabbitmq/issues/177.

By default, the bitnami Helm chart set the default already to `-`.

https://github.com/bitnami/charts/blob/531f7c28e5379fa8370cbc1bf8b23ac2ac486bb0/bitnami/rabbitmq/values.yaml#L150-L153

https://github.com/bitnami/charts/blob/531f7c28e5379fa8370cbc1bf8b23ac2ac486bb0/bitnami/rabbitmq/templates/statefulset.yaml#L164-L165

**Benefits**

The modification brings the possibility to setup more complex logs configuration via the `extraConfiguration` value from the Helm chart. Especially to set up JSON log format and an empty `RABBITMQ_LOGS` environment variable:
```
extraConfiguration: |-
  ...
  log.console.level = info
  log.console.formatter = json
  log.file = false
  ...

logs: ""
```

**Applicable issues**

#177

### CODE REVIEW

**Comment by aeimer** [comment]
@anthonyhaussman Thanks for fix! I just wanted to open a MR :D 

My thought was, do we even need to adapt the `RABBITMQ_LOGS` variable at all? The default value is set:
grafik
So from my perspective we could just remove the whole line + comment?!

**Comment by anthonyhaussman** [comment]
> My thought was, do we even need to adapt the `RABBITMQ_LOGS` variable at all? The default value is set: grafik So from my perspective we could just remove the whole line + comment?!

@aeimer Without the export, the default value will not be interpreted and will not be set to `-`.
And by default, RabbitMQ logs in file if `RABBITMQ_LOGS` is set without configuration.
An other solution will be to revamp the way the Helm chart setup the `log` configuration using templating.

**Comment by aeimer** [comment]
Hi @anthonyhaussman I guess it should work, as the default value is `logs: "-"` which will directly be set in the container:
grafik
So there is no need to export the variable again? Do I miss something?

**Comment by anthonyhaussman** [comment]
Indeed you are right @aeimer, I modify my PR :ok_hand:

File: 3.8/debian-10/rootfs/opt/bitnami/scripts/rabbitmq-env.sh [hunk 1]
```diff
@@ -49,7 +49,6 @@ rabbitmq_env_vars=(
     RABBITMQ_LDAP_SERVERS
     RABBITMQ_LDAP_SERVERS_PORT
     RABBITMQ_LDAP_USER_DN_PATTERN
-    RABBITMQ_LOGS
```
Comment (aeimer) [inline 3.8/debian-10/rootfs/opt/bitnami/scripts/rabbitmq-env.sh:52]: @anthonyhaussman not sure why this was removed. What does it do?

**Comment by javsalgar** [comment]
Thank you so much for the PR! 

My question is, if it is set to empty, I see that it would be overridden in the helm chart, but what happens in the docker-compose deployments we have in our repo?

**Comment by aeimer** [comment]
@javsalgar do you have a link to the files in the repo?

I guess you need to set some proper config file options or the envvar as needed/specified.

File: 3.8/debian-10/rootfs/opt/bitnami/scripts/rabbitmq-env.sh (hunk 1 above)
Comment (anthonyhaussman) [reply 3.8/debian-10/rootfs/opt/bitnami/scripts/rabbitmq-env.sh:52]: @aeimer It's used as a list to unset environment variable before re-set it if necessary:

https://github.com/bitnami/bitnami-docker-rabbitmq/blob/3e463f92e3b76983503422bbb764333e43f6e7b0/3.8/debian-10/rootfs/opt/bitnami/scripts/rabbitmq-env.sh#L78-L89

Not necessary here if we set it directly via the statefulsets.

**Comment by anthonyhaussman** [comment]
> Thank you so much for the PR!
> 
> My question is, if it is set to empty, I see that it would be overridden in the helm chart, but what happens in the docker-compose deployments we have in our repo?

@javsalgar I have updated docker-compose files to setup the `RABBITMQ_LOGS` environment variable by default. :slightly_smiling_face:

File: 3.8/debian-10/rootfs/opt/bitnami/scripts/rabbitmq-env.sh (hunk 1 above)
Comment (aeimer) [reply 3.8/debian-10/rootfs/opt/bitnami/scripts/rabbitmq-env.sh:52]: I see, thanks!

**Review by javsalgar — APPROVED** [review]

**Comment by javsalgar** [comment]
Awesome! I will work on integrating the changes in our automated pipeline :D

**Pull request merged**